tests/regression/echotest/ord-tests-expected.echo

```diff
@@ -1,26 +1,27 @@
 ECHO: 97
 ECHO: 97
+WARNING: ord() number of parameters does not match: expected 1, found 0 in file ord-tests.scad, line 9
 ECHO: undef
 ECHO: undef
-WARNING: ord() argument undef is not of type string, in file ord-tests.scad, line 11
+WARNING: ord() parameter could not be converted: argument 0: expected string, found undefined in file ord-tests.scad, line 11
 ECHO: undef
-WARNING: ord() argument inf is not of type string, in file ord-tests.scad, line 12
+WARNING: ord() parameter could not be converted: argument 0: expected string, found number in file ord-tests.scad, line 12
 ECHO: undef
-WARNING: ord() argument -inf is not of type string, in file ord-tests.scad, line 13
+WARNING: ord() parameter could not be converted: argument 0: expected string, found number in file ord-tests.scad, line 13
 ECHO: undef
-WARNING: ord() argument nan is not of type string, in file ord-tests.scad, line 14
+WARNING: ord() parameter could not be converted: argument 0: expected string, found number in file ord-tests.scad, line 14
 ECHO: undef
-WARNING: ord() argument 3.1416 is not of type string, in file ord-tests.scad, line 15
+WARNING: ord() parameter could not be converted: argument 0: expected string, found number in file ord-tests.scad, line 15
```

Comment:
Looks like this is not caused by your changes, but would you mind having a look if it's easily possible to get the actual value back into this message, that seems useful? Otherwise we can create a ticket so the topic is not lost.